CHANGELOG.md

```diff
@@ -234,22 +206,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Pinned typescript to ~4.7.x to address typing incompatibility issues
     discussed in https://github.com/knex/knex/pull/5279
   - Update generate-ts-build-cache script to always install root project dependencies
-
-### Fixed
```

Comment:
Same as above, just making sure these are intentional changes as they're out of scope for the 14.0 release.

Reply:
Correct in terms of this section was being restored to what should have been in [13.4.0](https://github.com/nasa/cumulus/releases/tag/v13.4.0)